packages/runner/lib/client.unit.test.ts

```diff
@@ -49,4 +49,9 @@ describe('Runner client', () => {
         const run = client.run.mutate({ nangoProps, isInvokedImmediately, isWebhook, code: jsCode });
         await expect(run).resolves.toEqual([1, 2, 3]);
     });
+
+    it('should cancel a sync', async () => {
+        const result = await client.cancel.mutate({ syncId: 'sync-id' });
+        expect(result).toEqual({ ok: true, res: 'cancelled' });
```

Comment:
~I'm surprised it works just a random syncId~
nevermind it cancels the previous test, might be cleaner to not depends on other test otherwise it will be hard to follow and can't be run independently for reproduction